app/routes/interestgroups/InterestGroupListRoute.js

```diff
@@ -15,9 +12,7 @@ const mapStateToProps = (state) => ({
   actionGrant: state.groups.actionGrant,
 });
 
-const mapDispatchToProps = { fetchAll, createInterestGroup };
-
 export default compose(
-  prepare((props, dispatch) => dispatch(fetchAll())),
-  connect(mapStateToProps, mapDispatchToProps)
+  prepare((props, dispatch) => dispatch(fetchAllWithType('interesse'))),
```

Comment:
Can we have this as a constant?

Reply:
Ouf, yeah, I guess so. Would be nice with importing types and constants from the backend tho.

Reply:
Did that nao